dist/install-common.sh

```diff
@@ -31,14 +27,14 @@ force_debug_mode() {
 }
 
 wait_for_atfwd_daemon() {
-    until [ -n "$(adb shell 'pgrep atfwd_daemon')" ]
+    until [ -n "$ADB shell 'pgrep atfwd_daemon'" ]
     do
         sleep 1
     done
 }
 
 wait_for_adb_shell() {
-    until adb shell true 2> /dev/null
+    until $ADB shell true 2> /dev/null
```

Comment:
it's usually good practice to wrap variables-in-commands in double quotes, otherwise all manner of chaos can happen if a space or `*` get into the variable. instead of find/replacing all instances of `adb` with `$ADB`, might be easier to make a `_adb_shell` function like i did with `_adb_push`